doc/source/api-reference/qibo.rst

```diff
@@ -744,6 +744,23 @@ Mølmer–Sørensen (MS)
     :members:
     :member-order: bysource
 
+Quantinuum native gates
+^^^^^^^^^^^^^^^^^^^^^^^
+
+U1q
+"""
+
+.. autoclass:: qibo.gates.U1q
+    :members:
+    :member-order: bysource
+
+.. note::
+    Note that the other Quantinuum single-qubit and two-qubit native gates are
```

Comment:
Maybe we can remove the redundancy
```suggestion
    The other Quantinuum single-qubit and two-qubit native gates are
```
since this is already a note.